frontend/src/main/components/Commons/CommonsPlay.css

```diff
@@ -0,0 +1,27 @@
+.radial-gradient-background {
+    background-image: radial-gradient(circle, #a3c3f0 0%, #64aa9e 100%); 
+    width: 100%;
+    height: 30vh;
+    justify-content: center;
+    display: flex;
+    color: #112731;
+    align-items: center;
+}
+
+/* .fade-in-text {
```

Comment:
Why is there commented out code?

https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

Reply:
I have fixed this and deleted the commented out code. Thank you.